platform-tests/src/test/java/org/junit/platform/launcher/core/LauncherDiscoveryRequestBuilderTests.java

```diff
@@ -199,36 +198,31 @@ void engineFiltersAreStoredInDiscoveryRequest() {
 		}
 
 		@Test
-		@SuppressWarnings("rawtypes")
 		void discoveryFiltersAreStoredInDiscoveryRequest() {
+			var filter1 = new DiscoveryFilterStub<>("filter1");
+			var filter2 = new DiscoveryFilterStub<>("filter2");
 			// @formatter:off
 			LauncherDiscoveryRequest discoveryRequest = request()
-					.filters(
-							new DiscoveryFilterStub("filter1"),
-							new DiscoveryFilterStub("filter2")
-					).build();
+					.filters(filter1, filter2)
+					.build();
 			// @formatter:on
 
-			List<String> filterStrings = discoveryRequest.getFiltersByType(DiscoveryFilter.class).stream().map(
-				DiscoveryFilter::toString).collect(toList());
-			assertThat(filterStrings).hasSize(2);
-			assertThat(filterStrings).contains("filter1", "filter2");
+			var filters = discoveryRequest.getFiltersByType(DiscoveryFilter.class);
+			assertThat(filters).containsOnly(filter1, filter2);
 		}
```

Comment:
Those test methods in `LauncherDiscoveryRequestBuilderTests` are much cleaner now. Thanks.